orquesta/machines.py

```diff
@@ -216,6 +235,8 @@
         events.WORKFLOW_FAILED: statuses.FAILED
     },
     statuses.FAILED: {
+        events.WORKFLOW_RERUNNING: statuses.RERUNNING,
+        events.TASK_RERUNNING: statuses.RERUNNING,
```

Comment:
Remove if these events are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.